modules/client/examples/index.ts

```diff
@@ -46,16 +46,24 @@ export const contextParams: ContextParams = {
   signer: new Wallet("private-key"),
```

Comment:
Should we have first an example with the very, very minimal working parameters?
Just a signer, the network and a free Web3 endpoint example.